packages/spruce-utils/lib/errors.js

```diff
@@ -5,6 +5,7 @@ function SpruceWebError(message = '', data) {
 	this.message = `${this.name} :: ${message}`
 	this.data = data
 	this.stack = new Error().stack
+	this.friendlyReason = data && data.friendlyReason
```

Comment:
Why can't this continue to live in `data`? Let's keep the top level clean if we can.